packages/fxa-content-server/app/tests/spec/views/mixins/password-mixin.js

```diff
@@ -187,13 +187,11 @@ describe('views/mixins/password-mixin', function () {
       it('getAffectedPasswordInputs - gets all affected inputs', function () {
         $('#container').html(view.$el);
         let targets = view.getAffectedPasswordInputs('#show-password');
-        // Note that this affects 5 in the test template, but with the real template,
```

Comment:
I was incorrect in my thinking here in working with the test template previously, and this was the problem. Too many inputs were affected.